src/HealthCheck/HealthCheckInterface.php

```diff
@@ -18,9 +18,12 @@
 
 namespace OpenConext\MonitorBundle\HealthCheck;
 
+use Symfony\Component\DependencyInjection\Attribute\AutoconfigureTag;
+
 /**
  * Contract for a HealthCheck.
  */
+#[AutoconfigureTag()]
```

Comment:
I feel this should be `#[AutoconfigureTag('surfnet.monitor.health_check')]`

Reply:
Can be, but if you skip the tag, Symfony will use the FQN of the interface. Awesome

bin/console deb:cont --tag="OpenConext\MonitorBundle\HealthCheck\HealthCheckInterface"

Symfony Container Services Tagged with "OpenConext\MonitorBundle\HealthCheck\HealthCheckInterface" Tag
======================================================================================================

 -------------------------------------------------------------------- -------------------------------------------------------------------- 
Service ID                                                           Class name
 -------------------------------------------------------------------- -------------------------------------------------------------------- 
OpenConext\MonitorBundle\HealthCheck\DoctrineConnectionHealthCheck   OpenConext\MonitorBundle\HealthCheck\DoctrineConnectionHealthCheck  
OpenConext\MonitorBundle\HealthCheck\SessionHealthCheck              OpenConext\MonitorBundle\HealthCheck\SessionHealthCheck
 -------------------------------------------------------------------- --------------------------------------------------------------------

Reply:
In that case, you should update the readme and state that those services are tagged with the FQN of the interface. And that is a BC break. Now upgrading users need to remove their userland tagged services and solely rely on the SF auto-tag logic.

Reply:
That actually is a good argument I did not think of. In that case, we'll re-introduce the tag, as it works fine with.

Only the code was a bit clearer to me just using the interface. But it is a small trade off for keeping BC

Reply:
Reverted back to using tags